tests/dummy/app/pods/components/frost-create-form-with-scroll/component.js

```diff
@@ -70,38 +79,24 @@ export default AbstractModal.extend({
     this.get('modalForms').setModalActive(false)
   },
 
```

Comment:
My mistake, so if you could please add...willDestroyElement needs to call super:
  /* Ember.Component method */
  willDestroyElement () {
    this._super(...arguments)
    this.get('modalForms').setModalActive(false)
  },